### PR TITLE
Create input pages for profile setup

### PR DESCRIPTION
**Pull request by iawale**

### CODE REVIEW

File: src/components/Birthday.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+const styles = StyleSheet.create({
+  body: {
+    flex: 1,
+  },
+  text: {
+    paddingTop: 16,
+    width: "80%",
+    fontSize: 16,
+  },
+  dateInput: {
```
[the remark this answers is not in the document]
Comment (iawale) [reply src/components/Birthday.tsx:56]: Bich ma bhanda tyai thik lagyyo, yo huda iphone ipad dubai size ma decent lagyo tyo demo ma hernubho?

**Review by bniroula — COMMENTED** [review]
Reviewed it but key the xaina react native ko :D

File: src/components/Birthday.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jshak) [reply src/components/Birthday.tsx:56]: I checked the demo. Actually not only date picker sabai page ma left tira aligned raicha mathy ko part and 'next' button chai center ma raichha which makes it look not aligned. Go ahead with this change.. we can revisit this later bro. Aru app ma heramla whats in trend.

**Review by rojan96 — APPROVED** [review]
Looks good aile lai, backend sanga connect garne ticket ni banam.

**Pull request merged**